tests/models/animals/conftest.py

```diff
@@ -34,46 +34,78 @@ def plant_climate_data_instance(fixture_core_components):
     grid = Grid(
         grid_type="square",
         cell_nx=3,
-        cell_ny=1,
+        cell_ny=3,
     )
     data = Data(grid)
 
-    leaf_mass = np.full((15, 3), fill_value=np.nan)
+    leaf_mass = np.full((15, 9), fill_value=np.nan)
```

Comment:
Would be better here to do:

```suggestion
    leaf_mass = np.full((fixture_core_components.layer_structure.n_layers, 9), fill_value=np.nan)
```

That then ties dimensions to the fixtures. When `grid` moves, we'd do the same.